### DISCUSSION TITLE
Custom "Getting Started" widget wont show in menu

### DISCUSSION THREAD

**Discussion by sboudreaux96** [eclipse-theia/theia#9679] [Q&A]
Hello - I have created a near-identical "getting started" widget that has very little changes (id and label are exactly the same as the theia getting started widget). I have noticed that my custom widget will not show under two conditions: 
1. If I dont use "@theia/getting-started" as a dependency it wont show in the menu. 
2. If I change the id of the getting started widget from "getting.started.widget" to "getting.started.widget2" it also doesnt work.

I would like to know what would cause something like this - thanks!

**Comment by vince-fugnitto** [comment]
@sboudreaux96 do you have the code somewhere where the community can better help you?
In addition, when starting your application if you see that commands or menus do not appear, it's likely you might encounter errors in the `dev console` which should help you identify the root cause.
